articles/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.conf import settings
+
+# Create your models here.
+class Article(models.Model):
+    title = models.CharField(max_length=30)
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, default="")
+    content = models.TextField(max_length=300)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    deleted_at = models.DateTimeField(null=True)
+    like_article = models.ManyToManyField(settings.AUTH_USER_MODEL, through="LikeArticle", related_name="like_article")
+
+class LikeArticle(models.Model):
```

Comment:
```suggestion
class LikeArticle(models.Model):
```
LikeArticle 是代表什麼？光看名稱有點看不懂

Reply:
這是文章與會員的第三方table
好我換個比較好的名稱